src/auth/services/auth.service.ts

```diff
@@ -37,6 +38,9 @@ export class AuthService {
   }
 
   async register(input: RegisterInput): Promise<RegisterOutput> {
+    // TODO : Setting default role as USER here. Will add option to change this later via ADMIN users.
+    input.roles = [ROLE.USER];
```

Comment:
@yashmurty 
LGTM. 
One question . You're setting `roles` to be `USER` in DTO . Why do you want to set roles here again ?

Reply:
Because anyone can overwrite the default role value set in the DTO. 🤔